badge_server/templates/one-badge.html

```diff
@@ -173,14 +173,14 @@ <h4>Google Compatibility Check Results</h4>
                         <ul class="nav nav-tabs" id="myTab" role="tablist">
                             <li class="nav-item">
                                 <a class="nav-link active" id="py2-tab" data-toggle="tab" href="#py2" role="tab" aria-controls="py2" aria-selected="true">Python 2
-                                {% if google_compat_res.py2.status == "CHECK_WARNING" %}
+                                {% if "INCOMPATIBLE" in google_compat_res.py2.status.name %}
```

Comment:
Why "in"?

Reply:
`CHECK WARNING` is now either `SELF INCOMPATIBLE` or `INCOMPATIBLE`

Reply:
I don't think that you should rely on that property.... on the other hand it is convenient. It's up to you.

Reply:
To give more context, this code inserts a 'warning' message if that statement is true, which looks like this:
![warning](https://user-images.githubusercontent.com/5758180/55654424-382a6180-57a6-11e9-8fd4-292a6e266387.png)

And I think refactoring the UI is beyond the scope of this PR (the goal is just to fix things that got broken).